### ISSUE TITLE
Reserve and Pay for Tee Times: Apple Pay

### ISSUE THREAD

**Issue by robinneb** [robinneb/golfzing#61]
## As a user I want to use Apple Pay to pay for my tee time. 

Dev - for Golfzing:

- Register a Merchant ID
- Generate a Payment Processing Certificate used to encrypt and decrypt payment tokens./li>
- Register and validate your website domain(if supporting Apple Pay on your website)
- Generate a Merchant Identity Certificate used to start an Apple Pay session and connect with Apple servers
- We will want to set up our server-side solution to receive payments & decrypt payment tokens and interface with the payment provider to process the authorization. 

Please see the attached example of the Apple Pay Flow

**Comment by drubage** [comment]
Again, I don’t see how this would be possible. EZLinks does not integrate with Apple Pay as far as I know to no-show charge people if they don’t show up.

[photo]

Drew Michael
VP, Technology, FRUITION
303.416.6943<tel:303.395.1880%20x23> | 303.931.5088<tel:303.931.5088> | drew@fruition.net<mailto:drew@fruition.net> | fruition.net<http://fruition.net/>
201 Fillmore Street Denver, CO 80206
[https://s3.amazonaws.com/images.wisestamp.com/icons/facebook.png]<http://www.facebook.com/fruition?fref=ts> [https://s3.amazonaws.com/images.wisestamp.com/icons/linkedin.png] <http://www.linkedin.com/company/134907?trk=tyah&trkInfo=clickedVertical%3Acompany%2Cidx%3A2-1-4%2CtarId%3A1428083749602%2Ctas%3Afruition>  [https://s3.amazonaws.com/images.wisestamp.com/icons/twitter.png] <http://twitter.com/FruitionSearch>

Awarded by Inc. 5000 as of one of the nation’s fastest growing private companies.

**Comment by robinneb** [comment]
BluePay who supports ApplePay would process the refund on behalf of EZLinks
as it does today with a credit card. I know this may seem like superfluous
work but mobile apps must have easy ways for users to transact without
putting in their credit card. I have built apps without Apple Pay or PayPal
and conversion was significantly lower without alternative payment
platforms. Also from a competitor standpoint, the other Golf apps have more
than one alternative to the credit card. Unless we have reviewed all of the
BluePay documentation and this is impossible, I'd like to leave in for the
client. It's worthwhile scope.
